Source/NSWindow.m

```diff
@@ -481,7 +481,7 @@ + (void) initialize
   
   iconSize = GSGetIconSize();
   
-  tileImage = [[GSCurrentServer() iconTileImage] copy];
+  tileImage = [[NSImage imageNamed:@"common_MiniWindowTile"] copy];
```

Comment:
Why would we use a different image here to the one used by NSApplication for the app icon?

Reply:
I attached images to PR. Miniwindow icon has different border around title (black background). With my change title doesn't draw background and shouldn't draw border around title to gain the same result as on screenshots.